extension/sql/meta.sql

```diff
@@ -20,6 +20,24 @@ GRANT SELECT ON ALL SEQUENCES IN SCHEMA vectorize TO pg_monitor;
 ALTER DEFAULT PRIVILEGES IN SCHEMA vectorize GRANT SELECT ON TABLES TO pg_monitor;
 ALTER DEFAULT PRIVILEGES IN SCHEMA vectorize GRANT SELECT ON SEQUENCES TO pg_monitor;
 
+CREATE OR REPLACE FUNCTION handle_table_drop() 
+RETURNS event_trigger AS $$
+DECLARE
+    obj RECORD;
+BEGIN
+    FOR obj IN SELECT * FROM pg_event_trigger_dropped_objects() LOOP
+        IF obj.object_type = 'table' AND obj.object_schema = 'vectorize' THEN
+            DELETE FROM vectorize.job 
+            WHERE name = obj.object_name;
```

Comment:
The tables that are dropped could be in any schema. And the record on the vectorize.job table that needs to be dropped points to that table in the params ->> 'table' and params ->> 'schema' values.

Reply:
yeah 😅